packages/labs/cli/src/lib/commands/init.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Command, CommandOptions} from '../command.js';
+import {run} from '../init/element-starter/index.js';
+import {LitCli} from '../lit-cli.js';
+
+export type Language = 'ts' | 'js';
+export interface InitCommandOptions {
+  lang: Language;
+  name: string;
+}
+
+export const makeInitCommand = (cli: LitCli): Command => {
+  return {
+    kind: 'resolved',
```

Comment:
I'm ok with this command being pre-installed, since we want people to get started with the lease ceremony and this doesn't seem like it adds much weight, but let's double-check with @kevinpschaaf and @rictic.

Reply:
All agreed offline.